src/search.js

```diff
@@ -284,6 +291,101 @@ export class SearchElement extends LitElement {
     `;
   }
 
+  renderRecentSearches() {
+    const recentSearches = this.getRecentSearches();
+    if (!recentSearches || !recentSearches.length) {
+      return html`<p>No Recent Searches</p>`;
```

Comment:
Unless this is to match for consistency, sentence case is a good fit here.

```suggestion
      return html`<p>No recent searches</p>`;
```